### ISSUE TITLE
Text Editor: Ctrl+arrow should skip words [windows]

### ISSUE THREAD

**Issue by ericroy** [defold/editor2-issues#477]
### Expected behaviour
On Windows, the Ctrl+Left and Ctrl+Right hotkeys should skip to the next word boundary

### Actual behaviour
The hotkeys currently skip to the beginning/end of the line.  I think this is the natural behaviour on mac, but not on windows.

### Steps to reproduce
Open a script component, and hit Ctrl+Left or Ctrl+Right

<hr/>

<table>
<tr><td>Defold version</td><td>1.2.97</td></tr>
<tr><td>Defold sha</td><td>e778f9ea025aa86ec65def2f3bf0ea0b4e7407f4</td></tr>
<tr><td>Build time</td><td>2017-02-09T17:34:29.812371</td></tr>
<tr><td>OS name</td><td>Windows 10</td></tr>
<tr><td>OS version</td><td>10.0</td></tr>
<tr><td>OS arch</td><td>amd64</td></tr>
<tr><td>Java version</td><td>1.8.0_102-b14</td></tr>
<table>

**Comment by Jerakin** [comment]
This is more annoying than one would think :|

**Comment by ericroy** [comment]
Yes, ultimately it's little things like this that are keeping me coding in an external editor...

**Comment by trevorade** [comment]
Came to look for this bug and found it.  It's seriously a crippling issue for me.

I would recommend supporting some sort of configuration file to store key-bindings and allow changing them.  This can be as clunky as you want to begin with and can later move into the editor as a GUI.

**Comment by AGulev** [comment]
Could you recheck it again pls? Is it still an issue? @ericroy @Jerakin

**Comment by Jerakin** [comment]
# Defold
Ctrl+Up: Go to start of document
Ctrl+Down: Go to end of document
Ctrl+Left: Go to first _word_ character on line
Ctrl+Right: Go to last character on line

Alt+Up: Go to first character on line
Alt+Down: Go to last character on line
Alt+Left: Skip word left
Alt+Right: Skip word right

# Other coding tools
Ctrl+Up: Scroll one line up (keep cursor position)
Ctrl+Down: Scroll one line down(keep cursor position)
Ctrl+Left: Skip word left
Ctrl+Right: Skip word right

Alt+Up: Go next function definition declared above
Alt+Down: Go next function definition declared below
Alt+Left: Cycle tabs left
Alt+Right: Cycle tabs right

## Bonus shortcuts that doesn't exists in Defold
Ctrl+Alt+Left: Go to last cursor position
Ctrl+Alt+Right: Go to previous cursor position

___

In short, the asked for shortcut exists but not on the hotkeys expected. I would argue that the standard is the standard and Defold should change theirs hotkeys around, at least the Ctrl+Left/Right. One could also argue for more hotkeys to be added (like cycle tabs and functions) but that's for a different feature request.

**Comment by vlaaad** [comment]
[Fixed](https://forum.defold.com/t/ux-improvements-for-code-navigation-and-keyboard-shortcuts/44967)!
